onnxruntime/core/session/provider_registration.cc

```diff
@@ -84,11 +84,7 @@ ORT_API_STATUS_IMPL(OrtApis::SessionOptionsAppendExecutionProvider,
     status = create_not_supported_status();
 #endif
   } else if (strcmp(provider_name, "OpenVINO") == 0) {
-#if defined(USE_OPENVINO)
```

Comment:
why is this removed?
It can break builds that don't have openvino enabled. e.g. https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1240589&view=logs&j=91d18614-091d-543c-f7ca-c2dad642e5d4&t=ce078c19-a5cd-536f-c923-6a49c8eb8ff8

Reply:
Hi George, This was a mistake and we've reverted these changes. However, we'll have to find an alternative soon to allow linking to signed OpenVINO dlls at runtime when the onnxruntime dlls weren't originally built with USE_OPENVINO.